st2common/st2common/services/datastore.py

```diff
@@ -90,7 +92,8 @@ def get_value(self, name, local=True, scope=SYSTEM_SCOPE, decrypt=False):
         try:
             params = {'decrypt': str(decrypt).lower(), 'scope': scope}
             kvp = client.keys.get_by_id(id=name, params=params)
-        except Exception:
+        except Exception, e:
```

Comment:
You can use Python >= 2.7 notation aka `except Exception as e` since we use it everywhere else.